src/ui/pages/History/AccelerateTransactionDialog/SpeedUp/SpeedUp.tsx

```diff
@@ -99,9 +100,9 @@ export function SpeedUp({
         { ...configuration, nonce: String(nonce) },
         acceleratedGasPrices
       );
+      assertProp(tx, 'chainId');
```

Comment:
Do we need the same in CancelTx?

Reply:
It didn't raise a type error so I didn't add this :)